packages/skygear-core/lib/database.js

```diff
@@ -33,6 +33,10 @@ export default class Database {
     this._cacheResponse = true;
   }
 
+  get Record() {
+    return this.container.Record;
+  }
```

Comment:
Well, what if this is for internal use? Like `User` and `publicDB` in `AuthContainer`. (I think these are two different cases, `User` can actually be imported directly, but `publicDB` cannot.)